udp_mux_multi.go

```diff
@@ -64,8 +148,18 @@ func (m *MultiUDPMuxDefault) Close() error {
 	return err
 }
 
-// GetListenAddresses returns the list of addresses that this mux is listening on
+// GetListenAddresses returns the list of addresses that this mux is listening on,
+// if port balance enabled and there are multiple mux listen on different ports of a same IP addr,
+// will return the mux who has least connections of that IP addr.
```

Comment:
```suggestion
// GetListenAddresses returns the list of addresses that this mux is listening on,
// if port balance enabled and there are multiple muxes listening to different ports of the same IP addr,
// it will return the mux that has the least number of connections.
```